common/formatting/layout_optimizer_internal.h

```diff
@@ -618,6 +621,33 @@ class LayoutFunctionFactory {
   const BasicFormatStyle& style_;
 };
 
+class TreeReconstructor {
+ public:
+  TreeReconstructor(int indentation_spaces, const BasicFormatStyle& style)
+      : current_indentation_spaces_(indentation_spaces), style_(style) {}
+  ~TreeReconstructor() = default;
+
+  // Delete standard interfaces
```

Comment:
(I think for deleting the copy constructors, we don't really need a comment. Unfortunately, C++ has the wrong default :/).
Up to you.

Reply:
Removed